SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -12,6 +12,7 @@
 #include "Interop/SpatialStaticComponentView.h"
 #include "Kismet/GameplayStatics.h"
 #include "Schema/AuthorityIntent.h"
+#include "Utils/InspectionColours.h"
```

Comment:
I thought we'd agreed to use `Color` and not `Colour`?

Reply:
ah my bad, I missed the filename!